assessments/projects/in-toto/self-assesment.md

```diff
@@ -0,0 +1,783 @@
+﻿# in-toto security self-assessment 
+## with input from the SIG-Security WG

```

Comment:
suggest dropping this second-line since it is implied by contributors list below